parastell/magnet_coils.py

```diff
@@ -238,6 +238,19 @@ def build_magnet_coils(self):
 
         self._cut_magnets()
 
+    def import_step_cubit(self):
+        """Import STEP file for magnet set into Coreform Cubit."""
+        if cubit_io.initialized:
+            first_vol_id = cubit.get_last_id("volume") + 1
+        else:
+            first_vol_id = 1
+
+        last_vol_id = cubit_io.import_step_cubit(
+            self.step_filename, self.export_dir
+        )
+
+        self.volume_ids = range(first_vol_id, last_vol_id + 1)
```

Comment:
Should this be a list?
```suggestion
        self.volume_ids = [range(first_vol_id, last_vol_id + 1)]
```